vzdump: add page (#7125)
tldr-pages/tldr: pages/common/vzdump.md

# vzdump

> Backup Utility for virtual machines and containers.
> More information: <https://pve.proxmox.com/pve-docs/vzdump.1.html>.

> - Dump a guest virtual machine into default dump directory (usually `/var/lib/vz/dump/`), no snapshot:

Reviewer: Perhaps this will read better?
```suggestion
- Dump a guest virtual machine into the default dump directory (usually `/var/lib/vz/dump/`), excluding snapshots:
```

`vzdump {{vm_id}}`

> - Backup guest 101, 102, and 103:

Reviewer: Is this an id number, a name, or something else?

Author: The number is the id of VM, in Proxmox, the VM start with 100, then 101

Author: ```suggestion
- Backup guest virtual machine with id 101, 102, and 103:
```

> - Backup guest virtual machine with id 101, 102, and 103:

Reviewer 2: `Backup` is only the noun, while `back up` is the verb
```suggestion
- Back up the guest virtual machines with the ID 101, 102, and 103:
```

Reviewer: `ID` → `IDs` @march

`vzdump {{101 102 103}}`

> - Dump a guest virtual machine using specific mode:

Reviewer: Perhaps this will read better?
```suggestion
- Dump a guest virtual machine using a specific mode:
```

`vzdump {{vm_id}} --mode {{suspend|snapshot}}`

> - Backup all guest systems and send notification mails to root and admin:

Reviewer: Perhaps this will read better?
```suggestion
- Backup all guest systems and send an notification email to the root and admin users:
```

> - Backup all guest systems and send an notification email to the root and admin users:

Reviewer 2: ```suggestion
- Back up all guest systems and send an notification email to the root and admin users:
```

`vzdump --all --mode {{suspend}} --mailto {{root}} --mailto {{admin}}`

> - Use snapshot mode (no downtime) and non-default dump directory:

Reviewer: Perhaps this will read better?
```suggestion
- Use snapshot mode (no downtime required) and a non-default dump directory:
```

> `vzdump {{vm_id}} --dumpdir {{/mnt/backup}} --mode snapshot`

Reviewer 3: Should this change? :thinking: 
```suggestion
`vzdump {{vm_id}} --dumpdir {{path/to/directory}} --mode snapshot`
```

> - Backup all guests excluding 101 and 102:

Reviewer: Same as above.

Reviewer 2: ```suggestion
- Back up all guest virtual machines excluding the ID 101 and 102:
```

Reviewer: Edited your suggestion @marchersimon to fix a typo - `ID` → `IDs`.

> `vzdump {{vm_id}} --dumpdir {{path/to/directory}} --mode snapshot`

Reviewer 2: ```suggestion
`vzdump {{vm_id}} --dumpdir {{path/to/directory}} --mode {{snapshot}}`
```
